test/unittest/StatisticsParticipantListener/StatisticsParticipantListenerTests.cpp

```diff
@@ -194,7 +198,8 @@ struct InsertEndpointArgs
                 const std::string& alias,
                 const Qos& qos,
                 const bool& is_virtual_metatraffic,
-                const eprosima::fastrtps::rtps::RemoteLocatorList& locators,
+                const eprosima::fastdds::
+                        rtps::RemoteLocatorList& locators,
```

Comment:
Apply to remaining cases
```suggestion
                const eprosima::fastdds::rtps::RemoteLocatorList& locators,
```